qa/L0_perf_deeprecommender/run_test.sh

```diff
@@ -70,6 +70,7 @@ for STATIC_BATCH in $STATIC_BATCH_SIZES; do
                         echo "dynamic_batching { preferred_batch_size: [ ${DYNAMIC_BATCH} ] }" >> config.pbtxt)
             fi
 
+            echo "Time before starting server: $(date)"
```

Comment:
Why do we need to emit these messages?

Reply:
> Print timestamps periodically to help debug timeouts

The shm and cudashm variants have timed out several times recently, so I wanted to add this to see what they're spending most of their time on when timing out in the future.